app/tests/spec/lib/fxa-client.js

```diff
@@ -17,8 +17,9 @@ define(function (require, exports, module) {
   var sinon = require('sinon');
   var testHelpers = require('../../lib/helpers');
 
+  var NON_SYNC_SERVICE = 'chronical';
```

Comment:
This used to be `'chronicle'` below and here it is `'chronical'`. Intentional or typo?